docs/install/install-serving-with-yaml.md

```diff
@@ -373,6 +375,6 @@ Refer to the "Real DNS" method for a permanent solution.
 
 After installing Knative Serving:
 
-- If you want to add extra features to your installation, see [Installing optional extensions](./install-extensions.md).
-- If you want to install the Knative Eventing component, see [Installing Eventing using YAML files](./install-eventing-with-yaml.md)
-- Install the [Knative CLI](./install-kn) to use `kn` commands.
+- [Installing Knative Eventing using YAML files](./install-eventing-with-yaml.md)
+
+- If you want to add extra features to your installation (like TLS), see [Installing optional extensions](./install-extensions.md).
```

Comment:
It seems weird to _just_ mention TLS here when talking about extra features. Maybe Just say "if you want to add TLS....see..."

Reply:
I bubbled up what I figured would be the most popular extension... maybe I'm better off just removing "(like TLS)"?

Reply:
Yeah I'd remove it

Reply:
Removed!